.github/workflows/playwright.yml

```diff
@@ -58,6 +58,17 @@ jobs:
           # prevent warnings if shard does not update any screenshots
           if-no-files-found: ignore
 
+      - name: Upload Reports
+        uses: actions/upload-artifact@v4
+        if: ${{ !cancelled() }}
```

Comment:
Should we only upload the reports if the pipeline failed?

Reply:
Is there a benefit of not uploading the reports for successful runs?

Reply:
I guess only saving disk space and keep the log minimal.
But I am also fine with keeping it like this